README.md

```diff
@@ -1,257 +1,9 @@
-
-\* \- Available on Linux OS only
-
-### Support for Hybrid Assembly
-
-To enable hybrid assembly, add the following to the configuration file.
-
-1. Add two read types to a sample:
-    - Paired-end reads
-    - Any type of long read
-2. Set the Unicycler assembly `"mode"` to `"hybrid"`.
-
-```
-{
-    "samples": {
-        "Shigella_sonnei_53G": {
-            "paired": [
-                ["yeat/tests/data/short_reads_1.fastq.gz", "yeat/tests/data/short_reads_2.fastq.gz"]
-            ],
-            "pacbio-corr": [
-                "yeat/tests/data/long_reads_high_depth.fastq.gz"
-            ],
-            "downsample": 0,
-            "genome_size": 0,
-            "coverage_depth": 150
-        }
-    },
-    "assemblies": {
-        "unicycler-hybrid": {
-            "algorithm": "unicycler",
-            "extra_args": "",
-            "samples": [
-                "Shigella_sonnei_53G"
-            ],
-            "mode": "hybrid"
-        }
-    }
-}
-```
-
-### Example config file
-
-```
-{
-    "samples": {
-        "sample1": {
-            "paired": [
-                [
-                    "yeat/tests/data/short_reads_1.fastq.gz",
-                    "yeat/tests/data/short_reads_2.fastq.gz"
-                ]
-            ],
-            "downsample": 0,
-            "genome_size": 0,
-            "coverage_depth": 150
-        },
-        "sample2": {
-            "paired": [
-                [
-                    "yeat/tests/data/Animal_289_R1.fq.gz",
-                    "yeat/tests/data/Animal_289_R2.fq.gz"
-                ]
-            ],
-            "downsample": 0,
-            "genome_size": 0,
-            "coverage_depth": 150
-        },
-        "sample3": {
-            "pacbio-hifi": [
-                "yeat/tests/data/ecoli.fastq.gz"
-            ],
-            "downsample": 0,
-            "genome_size": 0,
-            "coverage_depth": 150
-        },
-        "sample4": {
-            "nano-hq": [
-                "yeat/tests/data/ecolk12mg1655_R10_3_guppy_345_HAC.fastq.gz"
-            ],
-            "downsample": 0,
-            "genome_size": 0,
-            "coverage_depth": 150
-        }
-    },
-    "assemblies": {
-        "spades-default": {
-            "algorithm": "spades",
-            "extra_args": "",
-            "samples": [
-                "sample1",
-                "sample2"
-            ],
-            "mode": "paired"
-        },
-        "hicanu": {
-            "algorithm": "canu",
-            "extra_args": "genomeSize=4.8m",
-            "samples": [
-                "sample3"
-            ],
-            "mode": "pacbio"
-        },
-        "flye_ONT": {
-            "algorithm": "flye",
-            "extra_args": "",
-            "samples": [
-                "sample4"
-            ],
-            "mode": "oxford"
-        }
-    }
-}
-```
+YEAT documentation is available on the [YEAT GitHub wiki](https://github.com/rrwick/Bandage/wiki).
```

Comment:
Made the README significantly shorter and simpler. Added a link to the GitHub wiki page for installation instructions, settings, etc.

Reply:
Looks good. I'd just recommend pointing to YEAT's wiki instead of Bandage's wiki. 🙃